kudos-services/src/main/java/org/exoplatform/kudos/entity/KudosEntity.java

```diff
@@ -83,6 +83,9 @@ public class KudosEntity implements Serializable {
   @Column(name = "CREATED_DATE", nullable = false)
   public long               createdDate;
 
+  @Column(name = "SPACE_PRETTY_NAME")
+  private String            spacePrettyName;
```

Comment:
No need to store this information in fact, since you will use it only to generate activity in a dedicated stream

Reply:
Done!